### ISSUE TITLE
Detect already installed fonts

### ISSUE THREAD

**Issue by NathanBnm** [aimproxy/dotfonts#6]
I think you should add a way to detect if a font was already Installed.

For example I installed Roboto by clicking the "Install" button and it turned into "Installed" but after switching font or restarting the app I can Install it again.

The "Installed" button should remain if the font is already installed.

In the same idea it would be great to add a symbol that shows that the font is installed in the sidebar on the right of the font name. I'm thinking about a check mark or something.

**Comment by aimproxy** [comment]
Yeah, It's on my plans, and add a symbol that show the font is allredy installed on sidebar will be great, but a installed or downloaded stack will be great too!
